CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+Version 0.9.2 *(2019-11-16)*
+----------------------------
+
+ * Fix: Migrate to AndroidX.
```

Comment:
This is a fairly significant change for a patch version. Sounds like an 0.10.0 instead.